documents/customer.md

```diff
@@ -155,6 +155,249 @@ Customer customer = client.Customer.Fetch("cust_M462yViJrhNrQc");
 
 -------------------------------------------------------------------------------------------------------
 
+### Add Bank Account of Customer
```

Comment:
No updates in README.md file for this?

Reply:
fix doc. replaced `DeleteBankAccount` with `AddBankAccount`